Makefile

```diff
@@ -51,7 +51,15 @@ GO_BUILD       = env GOOS=$(PLATFORM) GOARCH=$(ARCH) go build -i $(GOFLAGS) \
 BASE_PATH      = $(ROOT:/src/github.com/kubernetes-incubator/service-catalog/=)
 export GOPATH  = $(BASE_PATH):$(ROOT)/vendor
 
-REGISTRY      ?= quay.io/kubernetes-service-catalog/
+MUTABLE_TAG                      = canary
```

Comment:
I can't see what the real world use case for that may be, but I also see no harm in it, so I will do that.